ambassador/src/test/java/com/iluwatar/ambassador/ClientTest.java

```diff
@@ -35,6 +37,6 @@ public void test() {
     Client client = new Client();
     long result = client.useService(10);
 
-    assert result == 100 || result == -1;
+    assertTrue(result == 100 || result == -1);
```

Comment:
We should use `RemoteServiceInterface.FAILURE` here as well. Also, we should create a contstant for 100 as well and use int respective tests.

Reply:
Oh yeah missed here. Done!
100 is calculated at runtime, so I guess we don't really need a constant as such.